docs/_posts/2014-01-06-community-roundup-14.md

```diff
@@ -30,13 +30,13 @@ React is only one-piece of your web application stack. [Mark Lussier](https://gi
 
 [Selem Delul](http://selem.im) bundled the [React Tutorial](/tutorial/tutorial.html) into a rails app. This is a good example on how to get started with a rails project.
 
+> ```sh
```

Comment:
Current markdown rendering engine requires these line to have a > at the start to treat the whole chunk as one single blockquote.